src/app/lib/constants/typography.js

```diff
@@ -0,0 +1,14 @@
+import { css } from 'styled-components';
+import mediaQuery from '../../helpers/mediaQueries';
+
+// Font styling below is a subset of BBC GEL Typography "Long Primer"
+const LONG_PRIMER = css`
+  font-size: 0.9375em;
+  line-height: 1.125rem;
+
+  ${mediaQuery.laptopAndLarger} {
+    font-size: 0.875em;
+  }
+`;
+
+export default LONG_PRIMER;
```

Comment:
Can you add a comment about what this file is, and how it distinguishes between `constants/styles`.